src/main/java/com/puppycrawl/tools/checkstyle/api/FileText.java

```diff
@@ -230,12 +229,7 @@ private static String readFile(final File inputFile, final CharsetDecoder decode
      */
     public static FileText fromLines(File file, List<String> lines) {
```

Comment:
It looks like all this does is call class constructor. Should we remove this and just call constructor directly?